dirs.proj

```diff
@@ -1,27 +1,52 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project=".\GlobalStaticVersion.props" />
```

Comment:
This comes from Global.props. One of the warnings during build was that GlobalStaticVersion.props was being imported multiple times, and removing this reference solved it.